src/Plugin/Action/AbstractGenerateDerivative.php

```diff
@@ -127,7 +141,7 @@ public function defaultConfiguration() {
       'mimetype' => '',
       'args' => '',
       'destination_media_type' => '',
-      'scheme' => file_default_scheme(),
+      'scheme' => $this->config->get('default_scheme'),
```

Comment:
Think with the change to `$this->config` we also need the configuration object passed in (`$this->config` doesn't already exist as part of the class)